test/tests/benchmark/utils/ansible/files/collect_data.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+log_path="/tmp/benchmark_log"
+interval=1
+atop_procid=`ps aux | grep atop | sed "/grep\|netatop/d" | awk '{print $2}' | sort -r -n`
+mongo_procid=`ps aux | grep mongo_disk.sh | sed "/grep/d" | awk '{print $2}' | sort -r -n`
+
+function terminate_process()
+{
+    # Kill existing atop process from the same user if any
+    if [ "$1" ]; then
+        for i in $1
+        do
+            kill -15 $i
+            echo "kill process $i"
+        done
+    fi
+}
+
+function start_atop_collection()
+{
+    terminate_process $atop_procid
+
+    # Get process PID
+    ps aux | grep -E "(mongod|beam.smp|on-|dhcpd)" | sed '/grep/d' | awk -F ' ' '{print $2, $11, $12, $13}' > ${log_path}/pid.log
+
+    # Get resource summary first
+    atop 1 1 -oa | grep "elapsed\|Kbps" | grep -v "NET" > ${log_path}/summary.log
+
+    # Monitor runtime data
+    nohup sh -c "atop $interval -oa | grep "Kbps" | grep -v "NET"  > ${log_path}/cpu_mem_net_disk.log" 2>/dev/null &
+}
+
+function stop_atop_collection()
+{
+    terminate_process $atop_procid
+
+    # Get resource summary at last
+    atop 1 1 -oa | grep "elapsed\|Kbps" | grep -v "NET" >> ${log_path}/summary.log
+}
+
+function start_mongod_collection()
```

Comment:
I am aware of this discussion in google group. I will change the name to make it more extensible like "start_db_collection". Once the data base changed, I could only update the subsequent shell scripts. Thanks.